docs/product/dev-toolbar/setup.mdx

```diff
@@ -70,20 +79,90 @@ And you can include any additional options from this list:
 | `mountPoint (optional)` | `HTMLElement \| () => HTMLElement \| undefined` | Where to mount the toolbar in the DOM. | `document.body` |
 
-## Implement Feature Flag Adapter
+### Unmounting The Toolbar
+
+If you have called `SentryToolbar.init({...})` to render the toolbar, but now want to manually remove or unmount it from the page, you can call the cleanup function that is returned from `init()`. This will unmount all the injected HTML and CSS. Login credentials will not be removed, so you can re-insert the toolbar and still be authenticated.
+```javascript
+const unmountToolbar = window.SentryToolbar.init({ ... });
+
+// sometime later...
+unmountToolbar();
+```
+
+## Deploying to Different Environments
+
+Since the Dev Toolbar will be visible to users within your app it's strongly recommended that you consider which environments the toolbar should apply to.
+
+In dev and staging environments include the toolbar so that all developers and testers can use it and quickly go from the page they're looking at back to Sentry for further debugging.
+
+In producion environments the Dev Toolbar can make it easier to reproduce real issues your users are having, but the toolbar should not be rendered for all users of the site, only when an employee/engineer/etc visits.
+
+
+It's possible to dynamically insert the script tag inside a single-page app, prior to calling `SentryToolbar.init()`, so that it’s only visible to authorized users, or in specific stages. See [`docs/conditional-script.md`](https://github.com/getsentry/sentry-toolbar/blob/main/docs/conditional-script.md) for example code. This will help reduce network traffic for your users because they do not have the credentials needed to login.
+
+The specific implementation for conditionally including the Dev Toolbar is something you'll need to write based on how your app works and how your dev team is set up. For example, the conditions to show show the toolbar in dev and staging might look like this:
+```typescript
+const env = process.env.SENTRY_ENVIRONMENT || 'development';
+
+const isDev = env === 'development' || env === 'staging';
+if (isDev) {
+  window.SentryToolbar.init({ ... });
+}
+```
+
+Or if your web application requires authentication to access, you could add a conditional where the Dev Toolbar is shown always when deployed to development **and** staging, but in production only show the toolbar **if** an employee is logged in.
+
+If the toolbar `<script>` is accidentally included on your site, and `SentryToolbar.init()` is called, then a "Login to Sentry" button will be visible to the public. This is not ideal, but your data in Sentry will still be safe as users outside of your Sentry organization will not be able to authenticate themselves.
+
+
+## Feature Flag Adapters
```

Comment:
Should we make this a more straight forward name like "Setting Up the Feature Flag Panel"? rather than referring the adapter technology

Reply:
I'm going to also link back to the FF issues stuff

Reply:
it's "Feature Flag Panel" now. 

And added an info:
>If you're using feature flags inside your product then also setup [Feature Flag Evaluation and Change Tracking](https://docs.sentry.io/platforms/javascript/feature-flags/) within your SDK.